### PR TITLE
Pin Terraform version

### PR DESCRIPTION
**Pull request by kumoy**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,6 +10,7 @@ ENV GOPATH=/go
 COPY --from=golang /go/ /go/
 COPY --from=golang /usr/local/go/ /usr/local/go/
 COPY --from=tfdocs /usr/local/bin/terraform-docs /usr/local/bin/
+COPY --from=terraform /usr/local/bin /usr/local/bin
```
Comment (lorengordon) [inline Dockerfile:13]: we don't want _all_ of /usr/local/bin, we only want the terraform binary....

```suggestion
COPY --from=terraform /usr/local/bin/terraform /usr/local/bin
```

Comment (kumoy) [reply Dockerfile:13]: So I actually have a question, I noticed we did that too with terraform-docs..
How do we know where the default installation destination is? I tried to make sense of the process in the Makefile but I don't think I got it...

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 FROM golang:1.16.2-buster as golang
-FROM quay.io/terraform-docs/terraform-docs:0.11.2 as tfdocs
+FROM quay.io/terraform-docs/terraform-docs:0.12.1 as tfdocs
```
Comment (lorengordon) [inline Dockerfile:2]:
```suggestion
FROM quay.io/terraform-docs/terraform-docs:0.11.2 as tfdocs
```

**Review by lorengordon — COMMENTED** [review]
Add a changelog entry and this looks good

**Review by lorengordon — APPROVED** [review]
Looks like you got it. On to the rest of the tools! 🚢

**Pull request merged**